qutebrowser/mainwindow/statusbar/percentage.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, parent=None):
         self.raw = False
 
     @pyqtSlot(int, int)
+    @throttle(100)
```

Comment:
This is a convenient place to put this, but if someone hypothetically overrode this class with their own, it would nullify the change.

I think I might be able to use the decorator to wrap the function before connecting it to `cur_scroll_perc_changed`, but I'm not sure if that's a better idea or not.